client/api/go-client/volume.go

```diff
@@ -232,3 +232,52 @@ func (c *Client) VolumeDelete(id string) error {
 
 	return nil
 }
+
+func (c *Client) VolumeClone(id string, request *api.VolumeCloneRequest) (*api.VolumeInfoResponse, error) {
```

Comment:
Ideally we can only keep `request` as in volume create(), we dont need `id` ( which is the source volume ID iic) and also `request` together. @phlogistonjohn @nixpanic  thoughts? or its required to have it compulsory for REST URL route?

Reply:
It seems like the new call simply follows the existing pattern (for example VolumeExpand). I don't see an issue with it as-is.